<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyInputDialog</class>
 <widget class="QDialog" name="MyInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加事件框</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="way_box">
      <item>
       <property name="text">
        <string>从不重复</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>每周</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>每月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>每x天</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="head_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入：</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>160</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="OK_button">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="input_line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>-1</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>从</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="from_date"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="to_date"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
